userspace/libsinsp/test/public_sinsp_API/interesting_syscalls.cpp

```diff
@@ -306,77 +117,24 @@ TEST(interesting_syscalls, io_sc_set)
 {
 	libsinsp::events::set<ppm_sc_code> io_sc_set_truth;
 
-#ifdef __NR_read
 	io_sc_set_truth.insert(PPM_SC_READ);
-#endif
-
-#ifdef __NR_recv
-	io_sc_set_truth.insert(PPM_SC_RECV);
-#endif
-
-#ifdef __NR_recvfrom
+	//io_sc_set_truth.insert(PPM_SC_RECV);
 	io_sc_set_truth.insert(PPM_SC_RECVFROM);
-#endif
-
-#ifdef __NR_recvmsg
 	io_sc_set_truth.insert(PPM_SC_RECVMSG);
-#endif
-
-#ifdef __NR_recvmmsg
 	io_sc_set_truth.insert(PPM_SC_RECVMMSG);
-#endif
-
-#ifdef __NR_readv
 	io_sc_set_truth.insert(PPM_SC_READV);
-#endif
-
-#ifdef __NR_preadv
 	io_sc_set_truth.insert(PPM_SC_PREADV);
-#endif
-
-#ifdef __NR_write
 	io_sc_set_truth.insert(PPM_SC_WRITE);
-#endif
-
-#ifdef __NR_pwrite
-	io_sc_set_truth.insert(PPM_SC_PWRITE);
-#endif
-
-#ifdef __NR_writev
+	//io_sc_set_truth.insert(PPM_SC_PWRITE);
```

Comment:
why is this commented? I don't see any `PPM_SC_PWRITE` between our enums, is it correct?

Reply:
Yep exactly; i don't know what that was indeed :)

Reply:
I think we can clean it up later!

Reply:
:woman_shrugging: I don't remember either and I was probably the one who added it? Plz cleanup and adjust as needed :pray:

Reply:
Cleaned up!